scripts/mktcli.js

```diff
@@ -130,36 +150,58 @@ const main = {
       )
 
     program
-      .command('publication <coord>')
-      .description('Get the current publication of a (x,y) coordinate')
+      .command('publication <assetId> <assetType>')
+      .description('Get the current publication of an asset')
       .action(
-        asSafeAction(async coord => {
-          const [x, y] = parseCLICoords(coord)
-          const id = Parcel.buildId(x, y)
-          const tokenId = await Parcel.encodeTokenId(x, y)
+        asSafeAction(async (assetId, assetType) => {
+          const asset = await getAssetFromCLIArgs(assetId, assetType)
 
-          const contract = eth.getContract('LegacyMarketplace')
-          const publication = await contract.auctionByAssetId(tokenId)
+          let contract = eth.getContract('LegacyMarketplace')
+          let publication = await contract.auctionByAssetId(asset.token_id)
 
-          const pubDb = (await Publication.findByAssetId(id))[0]
+          // publication[1] === owner. See `toPublicationLog`
+          if (Contract.isEmptyAddress(publication[1].toString())) {
+            const landRegistry = eth.getContract('LANDRegistry')
+            const estateRegistry = eth.getContract('EstateRegistry')
+
+            contract = eth.getContract('Marketplace')
+
+            switch (assetType) {
```

Comment:
If we use [have a helper function to get the contract based on the assetType](https://github.com/decentraland/marketplace/pull/588/files#diff-14c145377d6748040b73ee576c8e1d57R69)

then we can do
```javascript
const contract = getContractByAssetType(assetType)
const publication = await contract.orderByAssetId(
                  contract.address,
                  asset.token_id
                )
```